CONTRIBUTING.md

```diff
@@ -1,13 +1,35 @@
 # Contributing
 
-## Set up Git Hooks
-Run the following command to set up git hooks for the project. 
+## Prerequisites
+The following will need to be installed on your device in order to contribute to this project.
 
+- [Go >= v1.16](https://golang.org/dl)
+- [golangci-lint](https://golangci-lint.run/)
+- [Yarn](https://classic.yarnpkg.com/)
+
+## Development
+
+### Running Charmil CLI Commands
+You can run the CLI commands files directly with go run. All commands and subcommands are in the ./pkg/cmd folder.
+```bash
+go run ./cmd/charmil
 ```
-make setup/git/hooks
-```
 
-The following git hooks are currently available:
-- **pre-commit**:
-  - This runs checks to ensure that the staged `.go` files passes formatting and standard checks using gofmt and go vet.
-  
\ No newline at end of file
+### Development Commands
+
+#### `make setup/githooks`
+Run the command to set up git hooks for the project. The following git hooks are currently available:
+- **pre-commit** - This runs checks to ensure that the staged `.go` files passes formatting and standard checks using gofmt and go vet.
+
+#### `make lint`
+Runs a linter on the Go source code. Configuration can be found in .golangci.yaml. There are a number of lint rules enabled. You can find a full list of rules here with usage and configuration guides.
+
+#### `make test/unit`
+Runs unit tests
+
+## Community
+- We are keeping all the communications open, so that everyone can sync and is free to contribute. So if you have any feature/bugs suggestions about anything please donot hesitate to open up an [issue](https://github.com/aerogear/charmil/issues/new/choose)
```

Comment:
```suggestion
- We are keeping all the communications open, so that everyone can sync and is free to contribute. So if you have any feature/bugs suggestions about anything please don't hesitate to open up an [issue](https://github.com/aerogear/charmil/issues/new/choose)
```